mantidproject/mantid
# Update muon lifetime and ensure consistent throughout Mantid

Fixes #14977 

**Changes**
- Updated the muon lifetime stored in Mantid to the Particle Data Group's 2015 value (http://pdg.lbl.gov/2015/tables/rpp2015-sum-leptons.pdf). 
- Replaced several hard-coded instances of "2.2" with `Mantid::PhysicalConstants::MuonLifetime` value
- Updated unit tests to pass with new muon lifetime
- Updated doc tests for `CalMuonDeadTime`, `CalMuonDetectorPhases` and `RemoveExpDecay` 
- Updated system test for `CalMuonDeadTime`
- Updated [release notes](http://www.mantidproject.org/Release_Notes_3_6_MuonAnalysis)

**To test**
- Ensure all tests pass
- Code review: see if there are any locations missed
- Algorithms `CalMuonDeadTime`, `CalMuonDetectorPhases`, `PhaseQuad` and `RemoveExpDecay` will be affected - check these are working as they should
